nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -69,8 +69,10 @@ def __init__(self, aio_ctx: AioContext, connector: ConnectorInfo, conn_props: di
 
         conf = CommConfigurator()
         if conf.get_bool_var("simulate_unstable_network", default=False):
-            self.disconn = threading.Thread(target=self._disconnect, daemon=True)
-            self.disconn.start()
+            if context:
```

Comment:
maybe we could remove/move this testing related code to other place in future PRs

Reply:
Yes, this should become a general-purpose feature for all driver types.